lib/l10n/intl_en.arb

```diff
@@ -756,168 +756,286 @@
     "type": "text",
     "placeholders": {}
   },
-  "audit_action_message_create_workgroup": "{authorName} has created a new Workgroup : {resourceName}.",
-  "@audit_action_message_create_workgroup": {
+  "audit_action_message_create_workgroup_self": "You have created a new Workgroup : {resourceName}.",
```

Comment:
Why did you fix it like that? What if the `authorName ` passed in is sometimes not `You`. You should double check my code flow before fixing it. Be careful, errors will occur.